hera_objects/ObjectPosition[] objects_array